src/Hosting/Hosting/test/StaticWebAssets/StaticWebAssetsFileProviderTests.cs

```diff
@@ -117,6 +117,35 @@ public void StaticWebAssetsFileProvider_FindsFileWithSpaces()
             Assert.True(provider.GetFileInfo("/_content/Static Web Assets.txt").Exists);
         }
 
+        [Fact]
+        public void GetDirectoryContents_HandlesEmptyBasePath()
+        {
+            // Arrange
+            var provider = new StaticWebAssetsFileProvider("/",
+                Path.Combine(AppContext.BaseDirectory, "testroot", "wwwroot"));
+
+            // Act
+            var directory = provider.GetDirectoryContents("/Static Web/");
+
+            // Assert
+            Assert.Collection(directory,
+                file =>
+                {
+                    Assert.Equal("Static Web.txt", file.Name);
+                });
+        }
+
+        [Fact]
+        public void StaticWebAssetsFileProviderWithEmptyBasePath_FindsFile()
+        {
+            // Arrange & Act
+            var provider = new StaticWebAssetsFileProvider("/_content",
```

Comment:
How is `"/_content"` an empty base path?

Reply:
My bad, I think I undid something here. Thanks for catching it.

Reply:
Fixed